<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicStrokeWidgetForm</class>
 <widget class="QWidget" name="BasicStrokeWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_13">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QPushButton" name="m_strokeColorPushButton">
           <property name="text">
            <string>Color...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="m_strokeColorLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="m_strokeOpacityLabel">
           <property name="text">
            <string>Opacity %</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSlider" name="m_strokeOpacitySlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="m_strokeWidthLabel">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="m_strokeWidthDoubleSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="m_strokeUomComboBox"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="m_strokeDashLabel">
           <property name="text">
            <string>Dash:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="m_strokeDashComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="m_strokeJoinStyleLabel">
             <property name="text">
              <string>Join:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="m_miterJoinPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="m_roundJoinPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="m_bevelJoinPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="m_strokeCapStyleLabel">
             <property name="text">
              <string>Cap:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="m_buttCapPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="m_roundCapPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="m_squareCapPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_strokeColorPushButton</tabstop>
  <tabstop>m_strokeOpacitySlider</tabstop>
  <tabstop>m_strokeWidthDoubleSpinBox</tabstop>
  <tabstop>m_strokeUomComboBox</tabstop>
  <tabstop>m_strokeDashComboBox</tabstop>
  <tabstop>m_miterJoinPushButton</tabstop>
  <tabstop>m_roundJoinPushButton</tabstop>
  <tabstop>m_bevelJoinPushButton</tabstop>
  <tabstop>m_buttCapPushButton</tabstop>
  <tabstop>m_roundCapPushButton</tabstop>
  <tabstop>m_squareCapPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
